### ISSUE TITLE
NamedArguments with java code false positive

### ISSUE THREAD

**Issue by BraisGabin** [detekt/detekt#3289]
## Expected Behavior

This code should not raise any `NamedArguments` issue:
```kotlin
import org.joda.time.DateTime

DateTime(2020, 3, 13, 14, 0, 0)
```

## Observed Behavior
It does

## Context
If you use a java function in kotlin you can't use `NamedArguments` so there is no way to fix this.

## Your Environment
* Version of detekt used: 1.15.0-RC1

**Comment by schalkms** [comment]
Yes, that's true.
Currently, I don't think we have a rule that checks the target programming language of a called function.
For realizing this behavior, one needs to dig into the compiler and see, whether there is an API available.
If we are lucky, the `BindingContext` can be used to fetch this information.
